projects/demo/src/modules/directives/swipe/examples/2/index.ts

```diff
@@ -12,7 +12,7 @@ import {Subject} from 'rxjs';
     encapsulation,
 })
 export class TuiSwipeExample2 {
-    readonly open$ = new Subject();
+    readonly open$: Subject<boolean> = new Subject();
```

Comment:
Should be enough to type `new Subject<boolean>()`

Reply:
fixed